package.json

```diff
@@ -52,7 +52,7 @@
     "rollup-plugin-dts": "6.1.0",
     "rollup-plugin-preserve-shebangs": "0.2.0",
     "rollup-plugin-visualizer": "5.11.0",
-    "tsd-lite": "0.8.0",
+    "tstyche": "1.0.0-beta.6",
```

Comment:
You can also remove `@tsd/typescript` and `jest-runner-tsd`.

Reply:
Perfect, thanks